src/browser/client-scripts/build.js

```diff
@@ -0,0 +1,69 @@
+const path = require("path");
+const browserify = require("browserify");
+const uglifyify = require("uglifyify");
+const aliasify = require("aliasify");
+const fs = require("fs-extra");
+
+/**
+ * @param {object} opts
+ * @param {boolean} opts.needsCompatLib
+ * @returns {Promise<Buffer>}
+ */
+const bundleScript = async opts => {
```

Comment:
I am using plain JS with JSDoc instead of ts-node because ts-node is very slow, and it takes twice as more time to launch than running this script itself:
```bash
❯ time ./node_modules/.bin/ts-node -e ""
./node_modules/.bin/ts-node -e ""  3.76s user 0.26s system 143% cpu 2.803 total
```
```bash
❯ time node src/browser/client-scripts/build.js      
node src/browser/client-scripts/build.js  1.51s user 0.12s system 128% cpu 1.271 total
```